src/plugins/path/README.md

```diff
@@ -13,18 +13,18 @@ This plugin checks whether the value of a key is a valid filesystem path.
 ## Purpose ##
 
 The motivation to write this plugin is given by the two paths that exist
-in /etc/fstab: the device ﬁle and the mountpoint. A missing ﬁle is
-not necessarily an error, because the device ﬁle may appear later when
+in /etc/fstab: the device file and the mountpoint. A missing file is
+not necessarily an error, because the device file may appear later when
 a device is plugged in and the mountpoint may be there when another
 subsequent mount was executed. So only warnings are yielded in that
 case. One situation, however, presents an error: Only an absolute path
 is allowed to occur for both device and mount point. When checking for
-relative ﬁles, it is not enough to look at the ﬁrst character if it is
```

Comment:
urgh, thanks for fixing all this. Seems like someone did a copy and paste from a pdf...